.codegen/changelog_config.yml

```diff
@@ -0,0 +1,11 @@
+change_types:
+  - message: New Features and Improvements
```

Comment:
Matches the default. But in the future we can add a PR test to validate that all commits have a tag.